composer/lib/dependabot/composer/file_parser.rb

```diff
@@ -1,6 +1,7 @@
 # typed: true
```

Comment:
in the follow up; let's consider making this strong typing too

Reply:
Should I do that in this PR?

Reply:
It can be a separate PR to avoid blocking this one and for readability too.

Reply:
For sorbet typings another PR will be created.